TLC59116.h

```diff
@@ -524,6 +524,14 @@ class TLC59116Manager {
 
   friend class TLC59116;
 
+  private:
+  	// Moved and reorderd the following to remove a ton of [-Wreorder] compile warnings. 
+	// Specific bus
+    TwoWire &i2cbus;
+	long init_frequency;
```

Comment:
(note to self: fix indent)

Reply:
Looks like a tab was inserted (instead of spaces)